bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/internal/clangd/editor/ClangdConfigurationArea.java

```diff
@@ -64,13 +65,25 @@ public final class ClangdConfigurationArea {
 
 	private final Map<PreferenceMetadata<Boolean>, Button> buttons;
 	private final Map<PreferenceMetadata<String>, Text> texts;
+	private final Map<PreferenceMetadata<String>, Combo> combos;
 	private final List<Consumer<TypedEvent>> listeners;
 
+	private final static String[] completionOptions = { "detailed", "bundled", "" }; //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+	private final static String[] completionsKeys = { LspEditorUiMessages.LspEditorPreferencePage_completion_detailed,
```

Comment:
:-) Combo boxes often don't get translations because it is a bit of extra effort to code them properly. Thanks for taking the time to do that.

At some point after the first release we should make sure that the project gets added to eclipse babel. As such I created #174